test/integration/protocol_integration_test.cc

```diff
@@ -2436,4 +2436,26 @@ TEST_P(ProtocolIntegrationTest, RemoveResponseHeadersFilter) {
   EXPECT_THAT(response->body(), HasSubstr("missing required header: :status"));
 }
 
+TEST_P(ProtocolIntegrationTest, ReqRespSizeStats) {
+  config_helper_.addConfigModifier([&](envoy::config::bootstrap::v3::Bootstrap& bootstrap) -> void {
```

Comment:
can we add this modifier into (ProtocolIntegrationTest, Retry) and make sure we get both updates?

Reply:
Sure thing.